test/release/examples/primers/nothingVariables.chpl

```diff
@@ -1,32 +1,39 @@
-// Void variables and fields
+// The 'nothing' type
 
 /*
-   This primer demonstrates the use of the ``void`` type on variables and
-   class/record fields. A variable or field that is of type ``void`` is removed
-   by the compiler and doesn't exist during program execution.
+   This primer demonstrates the use of the ``nothing`` type on variables and
+   class/record fields. A variable or field that is of type ``nothing`` is
+   removed by the compiler and doesn't exist during program execution.
+
```

Comment:
I think the `none` value deserves a sentence (maybe in a standalone paragraph) at the top-level here for anyone trying to parse this file quickly.

Reply:
I added a sentence about `none`.